docs/Developers/Meta-review.rst

```diff
@@ -0,0 +1,140 @@
+ HEART        ``:heart:``    U+FE0F   ❤️
+============  =============  =======  =====
+
+These reactions are reviews to reviews, thus we call them meta-reviews. To
+encourage people to do meta-reviews, we build a meta-review system, which
+collects reactions, analyze them, and use that information to score people.
+After scoring, a ranking list is shown on coala community website.
+
+At the moment, only THUMBS_UP and THUMBS_DOWN are collected and analyzed,
+as they are most commonly used and express strong and clear feelings:
+THUMBS_UP means the review comment is well-written and helpful, while
+THUMBS_DOWN means the review comment is misleading, or even worse, violates
+`coala Community Code of Conduct <https://github.com/coala/cEPs/blob/master/cEP-0006.md>`__.
+
+Meta-review Process
+-------------------
+
+The meta-review process for coala is as follows:
+
+1. Reviewers create their reviews.
+
+2. Anyone especially author of the pull request can do meta-review (THUMBS_UP,
+   THUMBS_DOWN) via reactions on GitHub.
+
+3. Meta-review information will be collected and analyzed automatically:
+
+   * Meta-reviewers will be given bonus points to encourage such behaviour
+
+   * Meta-reviewees will be given positive points (on receiving THUMBS_UP) or
+     negative points (on receiving THUMBS_DOWN). Those points are weighted
+     according to score of meta-reviewers. Those who have higher score in
+     the meta-review system will have more impacts on others, so please do
+     meta-reviews carefully.
+
+.. note::
+
+    Don't edit or even delete your review after it has been meta-reviewed.
+    If you do that, you are destroying feedback from the community. This
+    is considered as an improper behaviour and a negative score will be
+    given to you.
+
+Automated Scoring Process
+-------------------------
+
+A complete ranking list can be found on
+`Meta-review score ranking list <https://deploy-preview-143--coala-community.netlify.com/meta-review/>`__.
+Score is based on number of positive (THUMBS_UP) & negative (THUMBS_DOWN)
+reactions one receives, and number of positive & negative reactions one gives
+away. Calculation details can be found in
+`meta_review/handler.py <https://github.com/li-boxuan/community/blob/2143fbcf925beb1ffe5ab1d9dd4d177e827dece2/meta_review/handler.py>`__.
+
+Briefly, score is calculated based on formulas as follows:
+
+Define:
+
+P1 = total points (weighted) of THUMBS_UP a person gets for all
+reviews he did.
+
+P2 = total number of THUMBS_UP a person gives to other
+people for their reviews.
+
+N1 = total points (weighted) of THUMBS_DOWN a person gets for all
+reviews he did.
+
+N2 = total number of THUMBS_DOWN a person gives to other people for
+their reviews.
+
+Then final score, denote by S, is as follows:
+
+S =  P1 - N1 + c1 * P2 + c2 * N2
+
+where c1 = 0.05, c2 = 0.2. One will get at least 0.1 point for a
+positive reaction they received, so we want c1 be smaller than that.
```

Comment:
c1 to* be smaller

Reply:
I am going to remove this part, but this review still helps as I made the same error in https://github.com/coala/community/pull/143 ;)